user_docs/en/changes.md

```diff
@@ -66,15 +65,12 @@ It currently includes Screen Curtain's settings (previously in the "Vision" cate
 
 * Fixed `<` not being escaped in MathML in PDF documents. (#18520, @NSoiffer)
 * When unicode normalization is enabled for speech, navigating by character will again correctly announce combining diacritic characters like acute ( &#x0301; ). (#18722, @LeonarddeR)
-* Fixed cases where NVDA was unable to retrieve information for an application, such as product name, version and architecture. (#18826, @LeonarddeR)
```

Comment:
please restore this: https://github.com/nvaccess/nvda/pull/18833#discussion_r2312675191